mos-platform/atari2600-3e/link.ld

```diff
@@ -0,0 +1,129 @@
+/* TigerVision 3E ROM mapper */
+
+INCLUDE vcs.ld
+
+PROVIDE(__cart_rom_size = 32);
+PROVIDE(__cart_ram_size = 32);
+
+ASSERT(__cart_rom_size >= 6,
+       "ATARI 2600 3E: ROM must be at least 6 KiB")
+ASSERT(__cart_rom_size <= 32,
+       "ATARI 2600 3E: ROM must be no larger than 32 KiB")
+ASSERT(__cart_rom_size % 2 == 0,
+       "ATARI 2600 3E: ROM must be multiple of 2 KiB")
+ASSERT(__cart_ram_size <= 255,
+       "ATARI 2600 3E: RAM must be no larger than 255 KiB")
+ASSERT(__cart_ram_size % 2 == 0,
+       "ATARI 2600 3E: RAM must be multiple of 2 KiB")
+
+/*
+   High word of origin address:
+   0x00 0x00    = fixed ROM
+   0x01 [bank#] = bank-switched ROM
+   0x02 [bank#] = bank-switched RAM
+*/
+
+MEMORY {
+  rom0 (rx)     : ORIGIN = 0x01001000, LENGTH = 0x800
```

Comment:
There's not much use setting these attributes, since these only control automatic placement of sections that aren't mentioned in the linker script. Due to the arcanity of those rules, we tend to require that all sections be explicitly mentioned in `SECTIONS`. In any case, only the first memory region with a given attribute set could possibly have any effect; there's no automatic spill-over once that one fills up; instead the link fails.